client/src/pages/signin/Signin.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from "react"
+import './Signin.scss'
+import { Link } from 'react-router-dom'
+
+import TextField from '@material-ui/core/TextField';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+
+class signin extends Component{
```

Comment:
Capitalize this - classes in Javascript are usually capitalized

Reply:
the class names have been updated